### PR TITLE
Renaming PrintQuality to PrintQuantity, Made 

### PR DESCRIPTION
**Pull request by comsma**
I did my tests with the few changes I made and everything works on a zt411 printer. Jimp 0.16.2 compared to 0.14.0 gives compiler errors when using it in my react v18.2 project.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@epickris/zpl",
-    "version": "0.2.1",
+    "version": "0.3.1",
```
Comment (SmolSoftBoi) [inline package.json:3]: Do not change the package version, this will be changed when a release is made.

File: src/index.ts [hunk 2]
```diff
@@ -2,7 +2,7 @@ import * as Commands from './commands';
 import * as CommandParams from './commandParams';
 
 import Jimp from 'jimp';
-import { padStart } from 'lodash';
+import {padStart} from 'lodash';
```
Comment (SmolSoftBoi) [inline src/index.ts:5]: Why the whitespace change here?

**Review by SmolSoftBoi — CHANGES_REQUESTED** [review]
Please make these changes.

File: package.json (hunk 1 above)
Comment (comsma) [reply package.json]: I just update this

File: src/index.ts (hunk 2 above)
Comment (comsma) [reply src/index.ts]: Typically I add padding. I just didn't have eslint setup

**Review by comsma — COMMENTED** [review]
Fixed package version and white space

**Review by comsma — COMMENTED** [review]
Fixed package version and white space

**Review by SmolSoftBoi — APPROVED** [review]

**Comment by comsma** [comment]
Thanks for merging that. Would you be able to publish the package to nom as well?

**Comment by SmolSoftBoi** [comment]
Could you let me know the Jimp error you were receiving, please? I want to figure out what's going on there.

**Comment by comsma** [comment]
I created a react project if you want to play around with it but this is the error I get with v16 of jimp. I am running node v19.0.1


[React demo app with @epickris/zpl package ](https://github.com/comsma/node-zpl-demo)
```
Module not found: Error: Can't resolve 'path' in '/Users/collinsmallegan/Projects/node-zpl/node_modules/@jimp/core/es'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
	- add a fallback 'resolve.fallback: { "path": require.resolve("path-browserify") }'
	- install 'path-browserify'
If you don't want to include a polyfill, you can use an empty module like this:
	resolve.fallback: { "path": false }
```

**Comment by SmolSoftBoi** [comment]
Ah yes, I recently ran across a similar issue, Webpack 5+ no longer includes poly fills, but you can fix the issue by following the instructions. Could you give this a go and let me know how you get on please?

**Pull request merged**